spec/index.html

```diff
@@ -780,28 +781,54 @@ <h3>Collections</h3>
+    <h3>Reified Triples</h3>
 
-    <p>A <dfn data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</dfn>
+    <p>A <dfn data-cite="RDF12-CONCEPTS#dfn-reified-triple">reified triple</dfn>
       may be the <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> or
       <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of an
       <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>.</p>
 
-    <p>A <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>
-      is represented using the <a href="#grammar-production-quotedTriple"><code>quotedTriple</code></a> production
-      with <a href="#grammar-production-subject"><code>subject</code></a>,
+    <p>A <a>reified triple</a> is syntactic sugar for a specific relationship between an
+      identifier (<dfn data-cite="RDF12-CONCEPTS#dfn-reifier">reifier</dfn>) and a <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>.
+      The identifier becomes a way to indirectly refer
+      to a <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>, which
+      may or may not be asserted within the graph corresponding to this input document.</p>
+
+    <p class="note">Reification in RDF 1.2 is a concept distinct from the
+      <a data-cite="RDF12-SEMANTICS#Reif">Reification vocabulary</a> originally
+      defined in <a data-cite="RDF-MT#Reif">RDF Semantics</a>.
+      While both terms describe a representation of an RDF triple using components,
+      RDF 1.2 uses the term to identify a <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>
+      using the `rdf:reifies` predicate.</p>
+
+    <p>A <a>reified triple</a> is represented using the
+      <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a> production
+      starting with <a href="#cp-double-lt"><code>&lt;&lt;</code></a>,
+      followed by <a href="#grammar-production-subject"><code>subject</code></a>,
       <a href="#grammar-production-predicate"><code>predicate</code></a>, and
-      <a href="#grammar-production-object"><code>object</code></a>
-      preceded by <a href="#cp-double-lt"><code>&lt;&lt;</code></a>, and
-      followed by <a href="#cp-double-gt"><code>&gt;&gt;</code></a>.
-      Note that <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triples</a>
-      may be nested.
-     </p>
-
-    <pre id="ex-quoted-triple"
+      <a href="#grammar-production-ttObject"><code>ttObject</code></a>,
+      optionally follwed by a <a href="#grammar-production-reifier"><code>reifier</code></a>
+      (<a href="#grammar-production-IRIREF"><code>IRIREF</code></a>
+      or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a>),
+      and ending with <a href="#cp-double-gt"><code>&gt;&gt;</code></a>.
+      For example, `&lt;&lt; :subject :predicate :object ~ :IRIREF &gt;&gt;`.
+      If the optional reifier is not present, a fresh RDF blank node is allocated,
+      as with `&lt;&lt; :subject :predicate :object &gt;&gt;`.</p>
```

Comment:
Not allowed by the grammar.

Reply:
There is no technical problem with `~` used as Ted suggests.  The usage of wantign to allocate the reifier id once when asserted for common use later has up before.

It has uses to allocate the id at that point so that there is one reifier agreed between later updates.

```
:s :p :o ~ .
```
```
INSERT  { ?e :added ?t } WHERE { :s :p :o ~?e . BIND(NOW() as ?t) }
```

Reply:
This comment is in the context of `reifiedTriple`, so I don't see a case where `<< :s :p :o ~ >>` does any more than `<< :s :p :o >>`, and is certainly not allowed by the grammar (at least the grammar that is included now).

In the annotation use case, `:s :p :o ~` could, indeed, allocate a new blank node, although one which cann't be referenced for creating more triples. It could be equivalent to the following:

```turtle
_:bn rdf:refies <<( :s :p :o )>> .
```

Where the `:_:bn` node is freshly allocated. Still not allowed in the existing grammar, and would require something like the following:

```bnf
annotation            ::= (('~' (iri | Blank Node)?) | '{|' predicateObjectList '|}')*
```

IMO, better to have a single `reifier` rule for both reified triples and annotations.

Reply:
In Turtle, yes, `<<:s :p :o >>` and `<<:s :p :o ~>>` amount to the same thing. 

The `:s :p :o ~` also asserts which is the preparation for later use is more motivating. It's more regularity to include it in `<<>>`.

> better to have a single reifier rule for both reified triples and annotations.

Agreed.